numba_cuda/numba/cuda/lowering.py

```diff
@@ -292,7 +292,7 @@ def lower_function_body(self):
         )
 
         # Lower all blocks
-        for offset, block in sorted(self.blocks.items()):
```

Comment:
If I'm reading this right, `self.blocks` was a `SortedMap`, and then we sorted it again before lowering. 😆